packages/runner-ct/static/index.html

```diff
@@ -7,20 +7,16 @@
       <meta name="viewport" content="width=device-width, initial-scale=1.0">
       <title>{{projectName}}</title>
 
-      <link href="/__cypress/runner/favicon.ico" rel="icon">
+      <link href="/{{namespace}}/runner/favicon.ico?v2" rel="icon">
 
-      <link rel="stylesheet" href="/__cypress/runner/cypress_runner.css">
+      <link rel="stylesheet" href="/{{namespace}}/runner/cypress_runner.css">
   </head>
   <body>
     <div id="app"></div>
-    <script type="text/javascript" src="/__cypress/runner/cypress_runner.js"></script>
+    <script type="text/javascript" src="/{{namespace}}/runner/cypress_runner.js"></script>
     <script type="text/javascript">
       // set a global so we know the 'top' window
       window.__Cypress__ = true
-
-      setTimeout(function(){
```

Comment:
This was pretty much 1:1 with `packages/runner/static/index.html`. Did we need to keep this timeout or do we need to remove the timeout in `packages/runner/static/index.html`.?